app/models/mandataris.js

```diff
@@ -94,6 +97,18 @@ export default class MandatarisModel extends Model {
     );
   }
 
+  get isInDraftStatus() {
+    // eslint-disable-next-line no-async-promise-executor
+    return new Promise(async (resolve) => {
+      const status = await this.publicationStatus;
+
+      if (status && status.uri === MANDATARIS_DRAFT_PUBLICATION_STATE) {
+        resolve(true);
+      }
+      resolve(false);
+    });
```

Comment:
You should just be able to solve this like this:
```suggestion
    return (
      this.publicationStatus?.get('uri') === MANDATARIS_DRAFT_PUBLICATION_STATE
    );
```